src/m365/spo/commands/folder/folder-add.ts

```diff
@@ -45,18 +56,32 @@ class SpoFolderAddCommand extends SpoCommand {
       },
       {
         option: '-n, --name <name>'
+      },
+      {
+        option: '--color [color]',
+        autocomplete: Object.entries(FolderColorValues).flat()
```

Comment:
Let's just use the color strings as numbers and leave away the numbers here. While numbers are correct values, it doesn't make sense to include them as autocomplete.